WORKSPACE

```diff
@@ -213,6 +213,7 @@ http_archive(
         # Diff is generated with a script, don't update it manually.
         "@mediapipe//third_party:org_tensorflow_custom_ops.diff",
         "tf.patch",
+        "tf_graph_info_multilinecomment.patch",
```

Comment:
There is commit by @michalkulakowski added today that fixes cpp benchmark app building. It also uses TF. Michal needed to bring all patches from OVMS to the benchmark app. Can we confirm this commit does not break cpp benchmark app again?

Reply:
Benchmark builds OK with this MR without additional changes.